README.md

```diff
@@ -1 +1,45 @@
 # puppet-healthcheck
+
```

Comment:
Right now this conflicts with the existing tcp_conn_validator code.  It'll need to be reworked.